package/kernel/linux/modules/ipmi.mk

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright (C) 2022 TDT AG <development@tdt.de>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See https://www.gnu.org/licenses/gpl-2.0.txt for more information
+#
+
+IPMI_MENU:=IPMI system interface support
+
+define KernelPackage/ipmi-handler
+  SUBMENU:=$(IPMI_MENU)
+  TITLE:=IPMI top-level message handler
+  KCONFIG:=CONFIG_IPMI_HANDLER=m \
+           CONFIG_ACPI_IPMI=m \
```

Comment:
This will generates separate acpi_ipmi.ko module. I think this would need packaging?

(ACPI_IPMI depends on CONFIG_IPMI_HANDLER, this could be used for AutoProbe over AutoLoad below)

Reply:
I'm not sure what you really mean here.
Do you think of another 'section' in the `ipmi.mk` with sommething like `define KernelPackage/acpi_ipmi` ... ?